styles.css

```diff
@@ -588,6 +588,61 @@ button:visited {
   color: #fff;
 }
 
+.union-vector .close-btn,
+.nav-toggle {
+  display: none;
+}
+
+.expand .nav-toggle {
+  margin-top: 108px;
```

Comment:
- Great job on this design 👍👌, I think You have a larger `margin-top` and different  font color compare to the Figma design, please stick to the Figma design, to have a better UI 👍👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/215717805-4456c218-6bab-4852-8231-6b809ddc5d1f.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/215717811-04577e2c-b069-4a23-9c7e-49a1aee45a71.png" width="200">|


😎️

🤝️